src/index.js

```diff
@@ -51,15 +50,25 @@ const insertTodos = () => {
     div.classList.add('item');
     div.appendChild(icon);
     div.appendChild(text);
-    div.appendChild(span);
+    div.appendChild(materiaIcon);
     div.style.order = todo.index;
```

Comment:
- [ ] When I delete an element, the id's to all lists could be updated, in this case, I could have index 0 and then index 1, but if you see the image below, the second item is still item 3. Please work on it. 🐶
- [ ] Hint: Please, it would be better if you use `id=` in place to put the id on the style tag. 
![image](https://user-images.githubusercontent.com/20563252/130080804-44735cad-8c4b-4f16-ac2a-dbb575f4aa8d.png)